### PR TITLE
added can_view_details and some of the columns. Changed the wallet_ad…

### PR DESCRIPTION
**Pull request by eoliveros**
…dress to information nad included bank account number in this column. Moved the acitve column nearer to the left side.

### CODE REVIEW

File: models.py [hunk 1]
```diff
@@ -662,7 +662,8 @@ def _format_address_column(view, context, model, name):
         asset_id = app.config["ASSET_ID"]
         explorer_url = app.config["BLOCKCHAIN_EXPLORER"]
         html = '''
-            <a href="{explorer_url}/address/{address}/tx">{address}</a>
+            wallet address: <a href="{explorer_url}/address/{address}/tx">{address}</a>
```
Comment (djpnewton) [inline models.py:665]: change function name to '_format_info_column'

File: models.py [hunk 2]
```diff
@@ -676,8 +677,9 @@ def _format_address_column(view, context, model, name):
                 }};
                 xhr.open("GET", url, true);
                 xhr.send();
-            </script>
-        '''.format(address=model.wallet_address, node_url=node_url, asset_id=asset_id, explorer_url=explorer_url)
+            </script><br/>
+            Bank Account number: {bank_account_number}
```
Comment (djpnewton) [inline models.py]: change label to 'bank account' (lowercase like the rest)

File: models.py [hunk 3]
```diff
@@ -691,11 +693,13 @@ def _format_bank_account(view, context, model, name):
...
         return Markup('-')
 
-    column_list = ['merchant_name', 'merchant_code', 'email', 'roles', 'confirmed_at', 'max_settlements_per_month', 'settlement_fee', 'merchant_rate', 'customer_rate', 'wallet_address', 'active', 'bank_account_number']
-    column_formatters = dict(wallet_address=_format_address_column, bank_account_number=_format_bank_account)
+    column_list = ['merchant_name', 'email', 'roles', 'active', 'max_settlements_per_month', 'settlement_fee', 'merchant_rate', 'customer_rate', 'information']
+    column_formatters = dict(information=_format_address_column, bank_account_number=_format_bank_account)
     column_filters = [ FilterStartsWithInsensitive(User.merchant_name, 'Search Merchant Name'), FilterStartsWithInsensitive(User.email, 'Search Email'), FilterBoolean(User.active, 'Filter Active') ]
     column_labels = {'bank_account_number': 'Bank AccountNumber(Active)'}
     column_formatters_export = {'wallet_address': _format_address_column_export, 'bank_account_number': _format_bank_account}
+    column_details_list = ['merchant_name', 'merchant_code', 'confirmed_at', 'bank_account_number']
```
Comment (djpnewton) [inline models.py]: should we have *all* the fields in the details view?

**Review by djpnewton — CHANGES_REQUESTED** [review]

**Comment by djpnewton** [comment]
cool did you test this?

**Comment by eoliveros** [comment]
> cool did you test this?

Yes, I did and looks good from this end.

**Pull request merged**